debian/control

```diff
@@ -0,0 +1,30 @@
+Source: quicktile
+Maintainer: Stéphane Gourichon <stephane_dpkg@gourichon.org>
+Section: misc
+Priority: optional
+Build-Depends: debhelper (>= 9),
+               build-essential,
+               devscripts,
+               dh-python,
+               fakeroot,
+               python3,
+               python3-pip,
```

Comment:
Does the Debian build process rely on `pip3` rather than going straight to `setup.py`? If not, then the `python3-pip` dependency is unnecessary and only included in the README listing to ensure that the recommended pip3-based installation process works.

Reply:
I tested removing this `python3-pip` on Debian (minideb flavour) and build failed. So I guess it is a necessary build-time dependency, probably due to Debian packaging program heuristics.

Reply:
Ahh. In that case, no problem leaving it.